modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -218,9 +218,14 @@ of {product-title} will support defining multiple compute pools during
 installation. Only one control plane pool is used.
 <3> Whether to enable or disable simultaneous multithreading (SMT), or `hyperthreading`. By default, SMT is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable SMT, you must disable it in all cluster machines; this includes both control plane and compute machines.
 +
+[NOTE]
+====
+Per default, if `hyperthreading` is enabled, ensure that simultaneous multithreading (SMT) is also enabled in your BIOS settings, on the contrary, the parameter has no effect.
```

Comment:
Could this be simplified a bit?
> If simultaneous multithreading (SMT) is not enabled in your BIOS settings, the `hyperthreading` parameter has no effect.

Reply:
Oh - and if you need the bit about default in there I suppose a sentence just saying "SMT is enabled by default" would be good :)